mon-pix/tests/unit/services/url_test.js

```diff
@@ -285,6 +285,105 @@ module('Unit | Service | url', function (hooks) {
     });
   });
 
+  module('#legalNoticeUrl', function () {
+    module('when website is pix.fr', function () {
+      test('returns the French page URL', function (assert) {
+        // given
+        const service = this.owner.lookup('service:url');
+        service.currentDomain = { isFranceDomain: true };
+        const expectedLegalNoticeUrl = 'https://pix.fr/mentions-legales';
+
+        // when
+        const legalNoticeUrl = service.legalNoticeUrl;
+
+        // then
+        assert.strictEqual(legalNoticeUrl, expectedLegalNoticeUrl);
+      });
+
+      module('when current language is "en"', function () {
+        test('returns the French page URL', function (assert) {
+          // given
+          const service = this.owner.lookup('service:url');
+          service.currentDomain = { isFranceDomain: true };
+          service.intl = { primaryLocale: ENGLISH_INTERNATIONAL_LOCALE };
+          const expectedLegalNoticeUrl = 'https://pix.fr/mentions-legales';
+
+          // when
+          const legalNoticeUrl = service.legalNoticeUrl;
+
+          // then
+          assert.strictEqual(legalNoticeUrl, expectedLegalNoticeUrl);
+        });
+      });
+
+      module('when current language is "nl"', function () {
+        test('returns the French page URL', function (assert) {
+          // given
+          const service = this.owner.lookup('service:url');
+          service.currentDomain = { isFranceDomain: true };
+          service.intl = { primaryLocale: DUTCH_INTERNATIONAL_LOCALE };
+          const expectedLegalNoticeUrl = 'https://pix.fr/mentions-legales';
+
+          // when
+          const legalNoticeUrl = service.legalNoticeUrl;
+
+          // then
+          assert.strictEqual(legalNoticeUrl, expectedLegalNoticeUrl);
+        });
+      });
```

Comment:
Question pour la team Accès : Est-ce que tester ces cas-ci (domain==pix.fr et language!==fr) est pertinent ?

Reply:
Ca me parait pertinent de tester ce cas oui !
Mais je pense qu'un seul test est suffisant avec par exemple comme titre `"when current language is not "fr""` parce que là les 2 tests testent le même bout de code.